src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -45,9 +48,11 @@ export class LocalGameWrapperComponent extends GameWrapper implements AfterViewI
         return MGPNodeStats.minimaxTime;
     }
     public ngAfterViewInit(): void {
-        this.afterGameIncluderViewInit();
-        this.restartGame();
-        this.cdr.detectChanges();
+        setTimeout(() => {
```

Comment:
Ah, il est de retour in fine ?
Qu'est ce qui a rétabli sa nécessité ?

Reply:
y'avait des tests cassés réparés par ce retour. Je pensais qu'il n'était pas nécessaire durant mon refactor (en tout cas pas pour un test à la main), mais visiblement bien nécessaire pour les tests